src/arrayMethodSort.js

```diff
@@ -5,7 +5,39 @@
  */
 function applyCustomSort() {
   [].__proto__.sort2 = function(compareFunction) {
-    // write code here
+    let callback = compareFunction;
+    let countArrayItterations;
+
+    if (arguments.length === 0) {
+      callback = (prev, curr) => {
+        if (String(prev) > String(curr)) {
+          return 1;
+        } else if (String(prev) === String(curr)) {
+          return 0;
+        }
+
+        return -1;
+      };
+    }
+
+    do {
```

Comment:
```suggestion
    const compareAsStrings = () => {};
    const callback = compareFunction || compareAsStrings;
    
    let countArrayItterations;

    do {
```
Create fallback function